library/core/src/ptr/unique.rs

```diff
@@ -201,3 +209,92 @@ impl<T: ?Sized> From<NonNull<T>> for Unique<T> {
         Unique { pointer, _marker: PhantomData }
     }
 }
+
+#[cfg(kani)]
+#[unstable(feature="kani", issue="none")]
+mod verify {
+    use super::*;
+
+    // pub const unsafe fn new_unchecked(ptr: *mut T) -> Self
+    #[kani::proof_for_contract(Unique::new_unchecked)]
+    pub fn check_new_unchecked() {
+        let mut x : i32 = 42;
```

Comment:
```suggestion
        let mut x : i32 = kani::any();
```

Reply:
why not use `kani::any()` in all these harnesses?

Reply:
Oh, sure, sorry. Done.